packages/api/test/post-build-test.js

```diff
@@ -32,10 +37,29 @@ describe('Posting Builds', () => {
       projectId: process.env.FLAKY_DB_PROJECT || 'flaky-dev-development'
     });
     global.headCollection = 'repositories-testsuite-' + uuidv4();
+
+    nock('https://api.github.com', {
```

Comment:
I use node fetch in my tests. I suppose consistency between different test files probably isn't very important, though.

Reply:
Nock is a mock http endpoint module so when you make a request to the github api to validate a token you dont need a real token. Instead you can use nock to make calling a node fetch to that url returns whatever you want.

Reply:
Oh okay I thought it was just an equivalent to node-fetch but that sounds awesome!